csv.h

```diff
@@ -49,11 +49,34 @@
 #include <cerrno>
 #include <istream>
 
+
+
 namespace io{
         ////////////////////////////////////////////////////////////////////////////
         //                                 LineReader                             //
         ////////////////////////////////////////////////////////////////////////////
 
+        template<char sep, bool null_terminated = true>
```

Comment:
null_terminated by default is true which makes no difference for existing user code.